backend/src/app/entities/sites.entity.ts

```diff
@@ -109,7 +109,16 @@ export class Sites extends ChangeAuditEntity {
   latdeg: number | null;
 
   @Field({ nullable: true })
-  @Column('double precision', { name: 'longdeg', nullable: true })
+  @Column('double precision', {
+    name: 'longdeg',
+    nullable: true,
+    transformer: {
+      // Convert positive longdeg values into negative (and therefore valid BC coordinates)
+      // In this context, `from` runs when reading from the database.
+      from: (value) => (value !== null && value > 0 ? -value : value),
```

Comment:
will this work for the existing data in the db? also if at all we have to sync back to oracle, we maybe have to write the reverse transformation

Reply:
>will this work for the existing data in the db?

yep

>also if at all we have to sync back to oracle, we maybe have to write the reverse transformation

If the sync will happen through the application layer and hit the Site entity we have declared here, then yes, we'll need to transform it back. Otherwise, no.